src/py/aspen/cli/db.py

```diff
@@ -168,3 +188,115 @@ def import_covidhub_project(
+
+        all_samples: Iterable[Sample] = (
+            session.query(Sample)
+            .filter(
+                or_(
+                    Sample.public_identifier.in_(samples),
+                    and_(
+                        expression.true()
+                        if all_group_sequences
+                        else expression.false(),
+                        Sample.submitting_group == group,
+                    ),
+                )
+            )
+            .options(
+                joinedload(Sample.sequencing_reads_collection)
+                .joinedload(SequencingReadsCollection.consuming_workflows)
+                .joinedload(Workflow.outputs)
+                .joinedload(Entity.consuming_workflows)
+                .joinedload(Workflow.outputs)
+                .joinedload(Entity.consuming_workflows)
+                .joinedload(Workflow.outputs),
+                joinedload(Sample.uploaded_pathogen_genome),
+            )
+        )
+
+        pathogen_genomes: MutableSequence[PathogenGenome] = list()
+        for sample in all_samples:
+            if sample.sequencing_reads_collection is not None:
+                sample_pathogen_genomes = (
+                    sample.sequencing_reads_collection.pathogen_genomes
+                )
+                # TODO: We are blindly using the first pathogen genome we encounter.  It
+                # should probably be based on some heuristic.
+                if len(sample_pathogen_genomes) > 0:
+                    pathogen_genomes.append(sample_pathogen_genomes[0])
+            elif sample.uploaded_pathogen_genome is not None:
+                pathogen_genomes.append(sample.uploaded_pathogen_genome)
+            else:
+                raise ValueError(
+                    "Sample without sequencing reads collection nor uploaded"
+                    " pathogen"
+                )
+        if len(pathogen_genomes) == 0:
+            raise ValueError("No sequences selected for run.")
+
+        if aligned_gisaid_dump_id is not None:
+            aligned_gisaid_dump = (
+                session.query(AlignedGisaidDump)
+                .filter(AlignedGisaidDump.entity_id == aligned_gisaid_dump_id)
+                .one()
+            )
+        else:
+            aligned_gisaid_dump = (
+                session.query(AlignedGisaidDump)
+                .join(AlignedGisaidDump.producing_workflow)
+                .order_by(Workflow.end_datetime.desc())
+                .first()
+            )
+
+        workflow: PhyloRun = PhyloRun(
+            start_datetime=datetime.datetime.now(),
+            workflow_status=WorkflowStatusType.STARTED,
+            software_versions={},
+            group=group,
+            # TODO: need build template / build args.
+        )
+        workflow.inputs = list(pathogen_genomes)
+        workflow.inputs.append(aligned_gisaid_dump)
+        workflow.template_file_path = (
+            "workflows/setup_nextstrain_run/builds_template.yaml"
```

Comment:
Where is this file?

Reply:
The workflow uses this file + the build args to generate the builds file.  It should be present in the target git refspec.  I have been iterating on the workflow in [this branch](https://github.com/chanzuckerberg/aspen/tree/ttung/setup-nextstrain-run).

Reply:
[here](https://github.com/chanzuckerberg/aspen/blob/ttung/setup-nextstrain-run/src/py/workflows/nextstrain_run/builds_template.yaml) in this case.